src/scripts/farming/WandererPokemon.ts

```diff
@@ -0,0 +1,49 @@
+class WandererPokemon {
+    public farmPoint: number;
```

Comment:
`farmPoint` doesn't appear to be used anywhere

Reply:
I forgot to remove it. Now it is done.